ResidentInformationApi/V1/Gateways/Helpers/DictionaryBuilder.cs

```diff
@@ -10,10 +10,10 @@ public static string BuildQueryDictionary(ResidentQueryParam rqp)
         {
             var queryDictionary = new Dictionary<string, string>();
 
-            if (!string.IsNullOrEmpty(rqp.FirstName)) queryDictionary.Add("first_name", rqp.FirstName);
-            if (!string.IsNullOrEmpty(rqp.LastName)) queryDictionary.Add("last_name", rqp.LastName);
-            if (!string.IsNullOrEmpty(rqp.Address)) queryDictionary.Add("address", rqp.Address);
-            if (!string.IsNullOrEmpty(rqp.Postcode)) queryDictionary.Add("postcode", rqp.Postcode);
+            if (!string.IsNullOrWhiteSpace(rqp.FirstName)) queryDictionary.Add("first_name", rqp.FirstName);
```

Comment:
Why are we changing it from `IsNullOrEmpty` to `IsNullOrWhiteSpace`?

Reply:
Good spot, this was changed for some testing, forgot to change it back, will do so now